src/main/java/usersandpostsCRUD/demo/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package usersandpostsCRUD.demo.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import usersandpostsCRUD.demo.dto.UserDto;
+import usersandpostsCRUD.demo.dto.UserResponseBody;
+import usersandpostsCRUD.demo.entity.User;
+import usersandpostsCRUD.demo.service.UserService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+@Autowired
+private UserService userService;
+
+    @GetMapping
+    public List<User> getAllUsers(){
```

Comment:
Return DTO everywhere, you should not expose the dabatase User to the end client.